src/components/results/doctorSpecialtiesListItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class DoctorSpecialtiesListItem extends React.Component {
```

Comment:
[comment] This would be a good example of an appropriate place to convert a Class component to a functional component. It does not have a need for its own state or life cycle methods.